_docs/reference/envvars.md

```diff
@@ -45,3 +45,8 @@ variables. These can be injected in via any of the usual mechanisms such as
 | `PROBE_INTERVAL_MS`       | 1.1.1             | The interval in milliseconds between node probes. Setting this lower (more frequent) will cause the cluster to detect failed nodes more quickly at the expense of increased bandwidth usage.  Defaults to 1000ms. Added in `1.1.1` and deprecated in `1.1.2` See `PROBE_INTERVAL`                             |
 | `PROBE_TIMEOUT`           | 1.1.2+            | The timeout to wait for an ack from a probed node before assuming it is unhealthy.  Takes a time duration in string format e.g. `500ms` or `2s`. This should be set to 99-percentile of RTT (round-trip time) on your network.  Defaults to 3000ms. Added in `1.1.2` replacing `PROBE_TIMEOUT_MS`             |
 | `PROBE_TIMEOUT_MS`        | 1.1.1             | The timeout to wait for an ack from a probed node before assuming it is unhealthy.  This should be set to 99-percentile of RTT (round-trip time) on your network.  Defaults to 3000ms. Added in `1.1.1` and deprecated in `1.1.2` See `PROBE_TIMEOUT`                                                         |
+| `TLSEtcdCA`               | 1.2.0             | The file path to the etcd server Certificate Authority certificate. The certificate must be mounted into the container using a ConfigMap or Secret. See [etcd TLS](/docs/operations/external-etcd/etcd-tls)                                                                                                                                                                                                                                           |
```

Comment:
Shouldn't this ENV variables be upper case?

Reply:
Agreed. Also, I suggest we remove the configmap reference - we shouldn't be encouraging users to present keying material using anything other than a secret.